data/blog/from-native-to-js.md

```diff
@@ -0,0 +1,103 @@
+---
+title: From Native to JS in Electron
```

Comment:
I'd suggest making the title a bit sharper – elaborate on `native` and use `JavaScript` instead of JS.